flask_debugtoolbar/panels/versions.py

```diff
@@ -3,13 +3,15 @@
 
 from flask import __version__ as flask_version
 from flask_debugtoolbar.panels import DebugPanel
+from flask import current_app
 
 _ = lambda x: x
 
 
 def relpath(location, python_lib):
     location = os.path.normpath(location)
-    relative = os.path.relpath(location, python_lib)
+    # relative = os.path.relpath(location, python_lib)
```

Comment:
Curious why you commented this out rather than removing it?

Reply:
Hi Jeff, you're right I should have just removed it. I left that in for my troubleshooting purposes, but forgot to remove it. My apologies.